### PR TITLE
Allow --text on Labels instead of --label.

### PR DESCRIPTION
**Pull request by erikcorry**

### CODE REVIEW

File: src/element.toit [hunk 1]
```diff
@@ -347,7 +347,9 @@ class Label extends Element implements ColoredElement:
     else if key == "alignment":
       alignment = value
     else if key == "label":
-      label = value
+      text = value
+    else if key == "text":
```
Comment (kasperl) [inline src/element.toit:351]: Maybe do `key == "text" or key == "label"` instead? Start with the preferred and keep the equivalent old version as the second check.

File: src/element.toit [hunk 2]
```diff
@@ -380,9 +382,11 @@ class Label extends Element implements ColoredElement:
   The $alignment is one of $ALIGN-LEFT, $ALIGN-CENTER, or $ALIGN-RIGHT.
   The $orientation is one of $ORIENTATION-0, $ORIENTATION-90, $ORIENTATION-180,
     or $ORIENTATION-270.
+  The $label argument is equivalent to the $text argument.  The $text
+    argument is preferred.
   The $color, $font, $orientation, and $alignment can be set using styles
-    instead of here in the constructor.  The label (text) can be set and
-    changed later with the label setter.  Like any change of appearance
+    instead of here in the constructor.  The $text can be set and
+    changed later with the text setter.  Like any change of appearance
```
Comment (kasperl) [inline src/element.toit:389]: Not sure if this works.

```suggestion
    changed later with the $(text=) setter.  Like any change of appearance
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/element.toit (hunk 2 above)
Comment (erikcorry) [reply src/element.toit:383]: sadly it doesn't.

**Pull request merged**